lib/navigation.js

```diff
@@ -2,6 +2,11 @@ const { dirname, join } = require('path')
 
 const { navigation } = require('../config')
 
+// Navigation item sorting function
+const { compare } = new Intl.Collator('en', {
```

Comment:
🆒

Reply:
I know we probably won't use it, but means can sort in natural English as/when we need to 😊